openapi/p4pa-classification.openapi.yaml

```diff
@@ -204,6 +204,27 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ClassificationErrorDTO'
+  /crud-ext/assessments/create-assessment-by-receipt-id:
+    post:
+      tags:
+        - assessment-entity-extended-controller
+      operationId: createAssessmentByReceiptId
+      parameters:
+        - name: receiptId
```

Comment:
what about accept the Receipt as body instead of requesting the id and search it?
CC: @marcomatteuccieng